README.md

```diff
@@ -45,13 +43,7 @@ Nebula Importer is compiled with Go **1.13** or later, so make sure that Go is i
 
 1. Clone the repository
 
-  * For Nebula Graph 1.x, clone the `v1` branch.
-
-  ``` bash
-  $ git clone --branch v1 https://github.com/vesoft-inc/nebula-importer.git
-  ```
-
-  * For Nebula Graph 2.x, clone the master branch.
+  * For Nebula Graph 3.x, clone the master branch.
```

Comment:
Are you sure for the master branch? If some years later...

Reply:
`README` should be updated per release and I think this is fine.